File: backend/src/main/java/com/livana/backend/reputation/dto/NgoLeaderboardEntryDto.java
package com.livana.backend.reputation.dto;

public record NgoLeaderboardEntryDto(
    String ngoAddress,
    /** Public display name of the verified NGO; null if not resolvable. */
    String orgName,
    long totalSbts,
    long totalAmountReleased,
    long poolCount,
    int rank
) {}

File: android/app/src/main/java/com/livana/app/core/network/dto/NgoLeaderboardEntryDto.kt
package com.livana.app.core.network.dto

import kotlinx.serialization.Serializable

@Serializable
data class NgoLeaderboardEntryDto(
    val ngoAddress: String,
    val orgName: String? = null,
    val totalSbts: Long,
    val totalAmountReleased: Long,
    val poolCount: Long,
    val rank: Int,
)
